src/execprogram.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EXECPROGRAM_H
+#define EXECPROGRAM_H
+
+#include "mostQtHeaders.h"
+
+class ExecProgram
+{
+public:
+	// Input parameters
```

Comment:
personally, I would like an constructor which allows setting the main parameters ...

Reply:
Added the constructors.